src/pages/games/[id]/edit.tsx

```diff
@@ -29,23 +31,23 @@ const EditGamePage = () => {
   const currentUser = session?.user;
   const [userData, setUserData] = useState<z.infer<typeof userDataSchema>>();
 
-  useEffect(() => {
-    if (!session) {
-      router.push("/");
-    }
-  }, [session]);
+  // useEffect(() => {
+  //   if (!session) {
+  //     router.push("/");
+  //   }
+  // }, [session]);
```

Comment:
ty for adding the edit page to the HOC component, you can delete these comments (and on line 46)!